src/layouts/EditPage.jsx

```diff
@@ -130,6 +130,20 @@ const EditPage = ({ match }) => {
         siteName,
         DOMCSPSanitisedHtml
       )
+
+      // NOTE: This is a hack to get around the fact that the DOMPurify library
+      // doesn't allow the <script> tag. Script sources are defined by our CSP
+      // policy, and inline-scripts are not allowed by our CSP policy as well.
+      // This will allow the user to save but not preview the script.
+      DOMPurify.removed = DOMPurify.removed.filter(
+        (el) =>
+          !(
+            el.element.tagName === "SCRIPT" &&
```

Comment:
btw this is case agnostic right? the `DOMPurify` repo seemed to have lowercase for `tagNames` (see [here](https://github.com/cure53/DOMPurify/blob/main/src/tags.js) for definitions and [here](https://github.com/cure53/DOMPurify/blob/main/demos/hooks-mentaljs-demo.html#L33) for their example where it's lower cased) but if you've tested this then that's fine

Reply:
Hmm not really, in this case the `el.element.tagName` attribute will have `SCRIPT` in uppercase, as the `el.element` object is the [native Element object](https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName).

> For DOM trees which represent HTML documents, the returned tag name is always in the canonical upper-case form. For example, `tagName` called on a &lt;div&gt; element returns `"DIV"`.

Reply:
interesting. the example given states that the dirty element is supposed to be html so i'm not too sure how this works as their example gives it as the lower-cased form ._. 

given that their definitions are in lower case, i think they might have abstracted this away from us and the casing doesn't actually matter. please resolve this if you've tested it and found it to work!